docs/src/pages/docs/closing-accounts-exploit.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Account Closing Exploit
+description: Anchor - In this Chapter of Common Security Exploits of Sealevel, or Sealevel Attacks, we do a deep dive on how malicious actors can use improper account closing to attack your program
+---
+
+## Overview
```

Comment:
No editorial / commentary in this one yet?

Reply:
correct, this is a bit above my pay-grade. I think it's still helpful to have all the code readable though.